engine/src/main/java/org/terasology/rendering/dag/stateChanges/EnableBlending.java

```diff
@@ -19,11 +19,16 @@
 import org.terasology.rendering.dag.StateChange;
 
 /**
- * Instances of this class enable OpenGL's blending, i.e. to render transparent objects or to make
- * a composite of different images.
+ * Enables blending in the OpenGL pipeline.
```

Comment:
I'd just write `Enables OpenGL Blending.` The term pipeline in our context can be a bit ambiguous I feel.